spec/std/crystal/compiler_rt/divmod128_spec.cr

```diff
@@ -0,0 +1,89 @@
+require "spec"
+
+# TODO: Replace helper methods with literals once possible
+
+private def make_ti(a : Int128, b : Int128)
+  (a << 64) + b
+end
+
+private def make_tu(a : UInt128, b : UInt128)
+  (a << 64) + b
+end
+
+# Specs ported from compiler-rt
```

Comment:
Can you add a direct reference to the origin here as well, please?
I think a permalink URL would be ideal.